src/Standards/Squiz/Tests/WhiteSpace/LanguageConstructSpacingUnitTest.inc

```diff
@@ -30,3 +30,10 @@ return;
 return $blah;
 return   $blah;
 return($blah);
+
+return	$tab;
+return
```

Comment:
You have right, but sniff description says:
> Ensures all language constructs contain a single space between themselves and their content.

And I think that it is exactly what the sniff should do, we didn't have covered that case previously in tests, so I think @gsherwood is gonna decide what to do.